### PR TITLE
Fix relative time filter 

### PR DESCRIPTION
**Pull request by laurakwhit**
## Description & motivation 💭 
* Update time range filter values when `Relative` is toggled on/off
* Converts relative time filter to a date to fix Invalid Date error
* Fixes position of datetime filter dropdown

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
- [ ] Verify the datetime filter dropdown is positioned correctly and does not go off the page

In Labs mode
- [ ] Verify a time range value is updated with `Relative` is togged on/off
- [ ] Verify selecting a relative time works as expected (and is converted to the timezone selected)

### CODE REVIEW

File: src/lib/components/workflow/filter-search/filter-list.svelte [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+  import { parseJSON } from 'date-fns';
```
Comment (rossedfort) [inline src/lib/components/workflow/filter-search/filter-list.svelte:2]: Unused import?

**Review by rossedfort — APPROVED** [review]

File: src/lib/components/workflow/filter-search/filter-list.svelte (hunk 1 above)
Comment (laurakwhit) [reply src/lib/components/workflow/filter-search/filter-list.svelte]: Yup, removed ✅

**Comment by rossedfort** [comment]
> When was it offscreen? When it's small screen, position right is cut off. Position left doesn't.

Yeah agree, I think we want this `position="left"`

**Comment by laurakwhit** [comment]
@Alex-Tideman I just fixed the position for the existing datetime filter dropdown in this PR ⬇️ 
|Before | After |
|--|--|
|Screenshot 2023-08-17 at 10 49 01 AM|Screenshot 2023-08-17 at 10 49 59 AM|

For the timezone dropdown responsiveness I believe @rossedfort has some goodness in the new combox PR.

**Pull request merged**